### ISSUE TITLE
Sigmoid warning

### ISSUE THREAD

**Issue by sleepinyourhat** [nyu-mll/jiant#322]
/home/sbowman/miniconda3/envs/jiant/lib/python3.6/site-packages/torch/nn/functional.py:1006: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
